src/main/resources/assets/mekanism/lang/zh_cn.json

```diff
@@ -724,6 +724,7 @@
   "fluid_container.mekanism.empty": "倒空",
   "fluid_container.mekanism.fill": "填充",
   "free_runner.mekanism.disabled": "禁用",
+  "free_runner.mekanism.safety": "安全性",
```

Comment:
this needs to also be added in `mekanism.client.lang.MekanismLangProvider` and datagen to be run

Reply:
OK, process later

Reply:
It has been fixed